docs/build/circuit-library.ipynb

```diff
@@ -0,0 +1,454 @@
+      "text/plain": [
+       "<Figure size 1207.22x284.278 with 1 Axes>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "bound_circuit = two_local.bind_parameters({ p: 0 for p in two_local.parameters})\n",
+    "bound_circuit.decompose().draw('mpl')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "78772993-d0da-4651-9211-706e86a59432",
+   "metadata": {},
+   "source": [
+    "For more information, see [N-local gates](/api/qiskit/circuit_library#n-local-circuits) in the circuit library API documentation or take our [Variational algorithm design course](https://learning.quantum-computing.ibm.com/course/variational-algorithm-design)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3e463081-bce9-4ee2-9046-97659e9ac847",
+   "metadata": {},
+   "source": [
+    "## Data encoding circuits\n",
+    "\n",
+    "These parameterized circuits encode data onto quantum states for processing by quantum machine learning algorithms. Some circuits supported by Qiskit are:\n",
+    " - Amplitude encoding, which encodes each number into the amplitude of a basis state. This can store $2^n$ numbers in a single state, but can be costly to implement.\n",
+    " - Basis encoding, which encodes an integer $k$ by preparing the corresponding basis state $|k\\rangle$.\n",
+    " - Angle encoding, which sets each number in the data as a rotation angle in a parameterized circuit.\n",
+    "\n",
+    "The best approach depends upon the specifics of your application. On current quantum computers, however, we often use angle encoding circuits such as the `ZZFeatureMap`."
```

Comment:
```suggestion
    "## Data-encoding circuits\n",
    "\n",
    "These parameterized circuits encode data onto quantum states for processing by quantum machine learning algorithms. Some circuits supported by Qiskit are:\n",
    " - Amplitude encoding, which encodes each number into the amplitude of a basis state. This can store $2^n$ numbers in a single state, but can be costly to implement.\n",
    " - Basis encoding, which encodes an integer $k$ by preparing the corresponding basis state $|k\\rangle$.\n",
    " - Angle encoding, which sets each number in the data as a rotation angle in a parameterized circuit.\n",
    "\n",
    "The best approach depends upon the specifics of your application. On current quantum computers, however, we often use angle-encoding circuits such as the `ZZFeatureMap`."
```